client/src/hooks/accountOperations.js

```diff
@@ -39,8 +37,8 @@ const useAccountOperations = () => {
     );
 
     return {
-        methods: { becomeTeacher, getAccount, getCourses, enrollCourse },
-        state
+        state,
```

Comment:
why is important for the state to be before methods

Reply:
it is not. I just do all imports in the same style